docs/developer/10-scm-plugins.md

```diff
@@ -23,6 +23,34 @@ Alternately, you can use the [Rundeck API - SCM](/api/rundeck-api.md#scm).
 SCM Configuration for a project is _not_ stored in the `project.properties`
 configuration contents.
 
+## Tuning 
+
+The following properties control how Process Automation interacts with external version control systems, such as Git or Subversion, to fetch and manage project definitions and job configurations. Here's a breakdown of each property:
```

Comment:
Let's specify where these can be set.  System Configuration and/or rundeck-config.properties.